website/docs/cli.md

```diff
@@ -98,6 +98,160 @@ against the prefix of these relative paths; others are substring matches.
+end
+
+example(1)
+```
+
+But if we pass the `--suggest-unsafe` flag to Sorbet, it will suggest an
+autocorrect:
+
+```
+❯ srb tc --suggest-unsafe
+editor.rb:6: Expected `Integer` but found `T.any(Integer, String)` for argument `arg0` https://srb.help/7002
+     6 |  1 + x
+              ^
+  Expected `Integer` for argument `arg0` of method `Integer#+`:
+    https://github.com/sorbet/sorbet/tree/master/rbi/core/integer.rbi#L148:
+     148 |        arg0: Integer,
+                  ^^^^
+  Got `T.any(Integer, String)` originating from:
+    editor.rb:5:
+     5 |def example(x)
+                    ^
+  Autocorrect: Use `-a` to autocorrect
+    editor.rb:6: Replace with `T.unsafe(x)`
+     6 |  1 + x
+              ^
+Errors: 1
+```
+
+[→ View without `--suggest-unsafe` on sorbet.run](https://sorbet.run/#%23%20typed%3A%20true%0Aextend%20T%3A%3ASig%0A%0Asig%20%7Bparams%28x%3A%20T.any%28Integer%2C%20String%29%29.void%7D%0Adef%20example%28x%29%0A%20%201%20%2B%20x%0Aend%0A%0Aexample%281%29)\
+[→ View with `--suggest-unsafe` on sorbet.run](https://sorbet.run/?arg=--suggest-unsafe#%23%20typed%3A%20true%0Aextend%20T%3A%3ASig%0A%0Asig%20%7Bparams%28x%3A%20T.any%28Integer%2C%20String%29%29.void%7D%0Adef%20example%28x%29%0A%20%201%20%2B%20x%0Aend%0A%0Aexample%281%29)
+
+Note how the autocorrect suggestion here suggests adding `T.unsafe`, but only
+when the `--suggest-unsafe` flag was provided.
+
+We recommend using `--suggest-unsafe` especially when doing Sorbet version
+upgrades. Usually the reason why a Sorbet version upgrade introduces a new error
+is because some old behavior was silently hiding a type error that was **already
+present** in the codebase. Using `T.unsafe` to silence the existing errors makes
+it easy to quickly upgrade to the new version, effectively locking newly-written
+code from suffering from that Sorbet bug. It's always possible to audit old
+usages of `T.unsafe`. Sorbet gets better with each release, and upgrading Sorbet
+early and often is usually less work over time.
+
+Sorbet also allows customizing the unsafe suggestions: given
+`--suggest-unsafe=<custom method>`, Sorbet will use the `<custom method>` in
+place of `T.unsafe` in all suggestions. For example:
+
+```
+❯ srb tc --suggest-unsafe=XXX.sorbet_0_5_1234 --autocorrect
+```
+
+This would cause case Sorbet to suggest converting `x` in the example above to
+`XXX.sorbet_0_5_1234(x)`. It would then be possible to define a module like
+this:
+
+```ruby
+module XXX
+  # Automatically-suppressed error during upgrade to Sorbet version 0.5.1234
+  #
+  # If you see this method used in a file your team owns, consider removing the
+  # call and fixing the resulting error, or replacing it with `T.unsafe` if the
+  # error cannot be fixed.
+  def self.sorbet_0_5_1234(x); x; end
+end
+```
+
+A method named like this sticks out in the codebase as a flag to potential
+developers that something has been automatically silenced vs been manually
```

Comment:
```suggestion
developers that something has been automatically silenced instead of being manually
```

This is less parallel, but avoids the "vs" abbreviation, which I think is marginally better.  Feel free to rewrite to a more parallel construction.